test/avocado/README.md

```diff
@@ -136,7 +136,7 @@ More concretely:
    stopped.
 
  * A test is a Python file in this directory modeled after
-   checkexample-foo.py.
+   example-check-foo.py.
```

Comment:
Can you please add a detailled commit message with the high-level description what this changes?